pkg/csconfig/common.go

```diff
@@ -11,7 +11,7 @@ import (
 /*daemonization/service related stuff*/
 type CommonCfg struct {
 	Daemonize    bool
-	PidDir       string     `yaml:"pid_dir"`
+	PidDir       string     `yaml:"pid_dir,omitempty"`
```

Comment:
maybe add a commz here so we don't forget about backward compat

Reply:
Done